README.md

```diff
@@ -176,9 +176,10 @@ Additional usage information is available from `manubot cite --help`:
 
 <!-- test codeblock contains output of `manubot cite --help` -->
 ```
-usage: manubot cite [-h] [--output OUTPUT] [--render]
-                    [--format {plain,markdown,docx,html,jats}] [--csl CSL]
-                    [--bibliography BIBLIOGRAPHY] [--allow-invalid-csl-data]
+usage: manubot cite [-h] [--output OUTPUT]
+                    [--format {csljson,cslyaml,plain,markdown,docx,html,jats} | --yml | --txt | --md]
```

Comment:
@agitter we could also do --cslyaml --yaml --plain --markdown. Here I went with most succinct, but it might be a bit confusion to not have format exactly match the shorthand.

Reply:
The way you set it up works for me.  It is slightly inconsistent, but since we're using it as shorthand I like the shorter versions (e.g. `--md` instead of `--markdown`).